src/NexusMods.App.UI/WorkspaceSystem/PanelResizer/PanelResizerView.axaml.cs

```diff
@@ -61,30 +81,25 @@ public PanelResizerView()
                 .Do(_ =>
                 {
                     _isPressed = false;
-                    _startPoint = new Point(0, 0);
+                    Icon.IsVisible = false;
```

Comment:
Don't set this to invisible here, it should only become invisible when the pointer exits.